crates/ev-cli/src/commands/interact.rs

```diff
@@ -38,32 +60,138 @@ pub mod validators {
             _ => Err(ValidationError::InvalidDestinationDomain),
         }
     }
+
+    pub fn validate_function_name(name: &str) -> Result<(), ValidationError> {
+        lazy_static::lazy_static!(
+            static ref NAME_REGEX: Regex = Regex::new(r"^[A-Za-z0-9]([-_]?[A-Za-z0-9])*$").unwrap();
+        );
+        let length = name.len();
+        let regex_result = NAME_REGEX.is_match(name);
+        if !((length >= 2) & (length <= 40) & regex_result) {
+            return Err(ValidationError::InvalidFunctionName);
+        }
+        Ok(())
+    }
+
+    pub fn validate_function_language(language: &str) -> Result<(), ValidationError> {
+        lazy_static::lazy_static!(
+            static ref LANGUAGE_REGEX: Regex = Regex::new(r"^node@18|node@20|python@3\.9|python@3\.10|python@3\.11$").unwrap();
```

Comment:
Should we strip the specific versions here? Consistent with the API validation approach